microprofile-rest-client/README.adoc

```diff
@@ -50,6 +50,7 @@ include::../shared-doc/run-arquillian-tests.adoc[leveloffset=+1]
 
 // Run the Quickstart in Red Hat CodeReady Studio or Eclipse
```

Comment:
- please remove this include of deprecated section
- please remove https://github.com/wildfly/quickstart/pull/780/files#diff-585a2749bf0bbb7eeed651cb57466a95940c6e603b8572ba3b7289a87941581eL48
- at https://github.com/wildfly/quickstart/pull/780/files#diff-585a2749bf0bbb7eeed651cb57466a95940c6e603b8572ba3b7289a87941581eL44 please add
// Server Distribution Testing
include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+1]